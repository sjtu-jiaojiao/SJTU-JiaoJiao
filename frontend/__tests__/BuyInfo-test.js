import React from 'react';
import BuyInfoScreen from '../src/Views/BuyInfo/BuyInfo';

import renderer from 'react-test-renderer';

test('renders correctly', () => {
    const tree = renderer.create(<BuyInfoScreen />).toJSON();
    expect(tree).toMatchSnapshot();
});
