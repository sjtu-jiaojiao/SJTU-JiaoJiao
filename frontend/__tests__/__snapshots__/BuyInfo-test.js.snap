// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View>
  <Text>
    浏览求购信息
  </Text>
</View>
`;
